base/interactiveutil.jl

```diff
@@ -598,7 +598,7 @@ else
                 rethrow()
             end
         elseif downloadcmd == :curl
-            run(`curl -L -f -o $filename $url`)
+            run(`curl -L -f -s -o $filename $url`)
```

Comment:
you've seen https://github.com/JuliaLang/BinDeps.jl/issues/135, but this function could probably use a more serious refactoring - not everyone will want all downloads to be silent. you can use curlrc to control some of these globally if you know you're using curl